### DISCUSSION TITLE
[Question] speaker.volume() method support on Robot Inventor hub ?

### DISCUSSION THREAD

**Discussion by johnscary-ev3** [pybricks/support#670] [Q&A]
It appears that speaker.volume() method is supported in the "latest" API documentation.
However after getting the latest install from install.pybricks.com, I don't get this method.
Probably I am not actually getting latest firmware, or this method is not actually supported yet.
I have this firmware version below. If not "latest", how do I get that installed?   Thanks much.

v3.1.0-110-gfedf09ed on 2022-01-17

**Comment by dlech** [comment]
Please see the recent discussion in https://github.com/pybricks/support/issues/626. It also applies to the Robot Inventor hub. TL;DR; is to grab the latest firmware build from github actions and use `pybricksdev` to flash the firmware.

**Comment by johnscary-ev3** [comment]
Thanks for the reply.
After reading #626, I grabbed the latest firmware and installed latest pybricksdev.
Seems like there are a few bugs using pybricksdev with respect to different revisions of Python. I see same errors reported in #626.
I currently have Py 3.8 since that is what was supported previously.  Now looks like it may need version 3.10 ?
Can you confirm the Python version we should be using now?
Looking at the info in #626, seems like some features of Py 3.10 are used now which created problems for 3.8 but you went back and are fixing those but have not released those fixes yet. Is that correct?
Thanks much for the help.

**Reply by dlech** [reply]
Yes, that is correct. I guess we better do a release with the fixes.
